src/components/global/nav-bar/nav-button-container.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import BaseButton from '../base-button.vue';
```

Comment:
Use `@` path to point to `base-button.vue`

File structure might change and at least you won't have to deal with imports inside files that changes its own path : )

Reply:
Good point, I updated the paths throughout the project.